### PR TITLE
Rand pie

### PR DESCRIPTION
**Pull request by RandyHuynh5815**
Pie moment

### CODE REVIEW

**Comment by parsakmt** [comment]
I like that pie

File: Piethon.py [hunk 1]
```diff
@@ -0,0 +1,7 @@
+def pitime(pie):
+    if (pie == 3.1415):
+        return 'Celebrate Pi Time'
+    return 'Sad no pi moment'
```
Comment (me-here) [inline Piethon.py:4]: happy pi! nice work

**Review by parsakmt — COMMENTED** [review]
Very nice pie 10/10

**Review by me-here — COMMENTED** [review]
perfect

**Review by parsakmt — APPROVED** [review]
Amazing

**Review by me-here — APPROVED** [review]

**Pull request merged**